src/sidebar/store/modules/groups.js

```diff
@@ -122,14 +170,33 @@ function focusedGroupId(state) {
 }
 
 /**
- * Return the list of all groups.
+ * Return the list of all groups, ignoring any filter present.
```

Comment:
In the previous version of these changes, `allGroups` would filter out `filteredGroups`. These changes leave it as-is but create a `filteredGroups` selector (which is referenced by the other group-fetching selectors).